frontend/package.json

```diff
@@ -17,6 +17,7 @@
     "nock": "^13.2.4",
     "react": "^17.0.1",
     "react-bootstrap": "^2.0.0-alpha.0",
+    "react-datepicker": "^7.5.0",
```

Comment:
Where is this used in the code?  If it is not *used*, take it out.

Reply:
I am sorry that this line appears here. I did not use it, and I really did not add it by myself. I may accidentally pull it from Yixuan's code. I will delete it.